roles/fio_distributed/templates/server_vm.yml.j2

```diff
@@ -22,60 +22,70 @@ spec:
       cores: {{ workload_args.vm_cores | default(1) }}
     devices:
       disks:
-        - name: registrydisk
-        - name: cloudinitdisk
+        - disk:
+            bus: {{ workload_args.vm_bus | default('virtio') }}
+          name: registrydisk
+        - disk:
+            bus: {{ workload_args.vm_bus | default('virtio') }}
+          name: cloudinitdisk
 {% if workload_args.storageclass is defined
     or workload_args.hostpath is defined %}
         - disk:
-            bus: virtio
+            bus: {{ workload_args.vm_bus | default('virtio') }}
           name: data-volume
           serial: data
+{% else %}
+        - disk:
+            bus: {{ workload_args.vm_bus | default('virtio') }}
+          name: emptydisk
 {% endif %}
     resources:
       requests:
         memory: {{ workload_args.vm_memory | default('5G') }}
   volumes:
     - name: registrydisk
       containerDisk:
-        image: {{ workload_args.vm_image | default('kubevirt/fedora-cloud-container-disk-demo:latest') }}
+        image: {{ workload_args.vm_image | default('quay.io/kubevirt/fedora-cloud-container-disk-demo:latest') }}
```

Comment:
@jeniferh, Do you think we need to update it to the new fedora image ?
`quay.io/kubevirt/fedora-container-disk-images:34`

Reply:
Good point, I will leave the operator tag as `:latest` using this newer image, and we can let the runner be opinionated using the `:34` tag and moving up when appropriate.

Reply:
OK, sounds good
/LGTM